src/components/shared/CallNodeContextMenu.js

```diff
@@ -502,16 +515,32 @@ class CallNodeContextMenu extends PureComponent<Props> {
 }
 
 export default explicitConnect<{||}, StateProps, DispatchProps>({
-  mapStateToProps: state => ({
-    thread: selectedThreadSelectors.getFilteredThread(state),
-    threadIndex: getSelectedThreadIndex(state),
-    callNodeInfo: selectedThreadSelectors.getCallNodeInfo(state),
-    implementation: getImplementationFilter(state),
-    inverted: getInvertCallstack(state),
-    callNodePath: selectedThreadSelectors.getRightClickedCallNodePath(state),
-    callNodeIndex: selectedThreadSelectors.getRightClickedCallNodeIndex(state),
-    selectedTab: getSelectedTab(state),
-  }),
+  mapStateToProps: state => {
+    const threadIndex = getRightClickedCallNodeThreadIndex(state);
+    const callNodePath = getRightClickedCallNodePath(state);
+    const callNodeIndex = getRightClickedCallNodeIndex(state);
+    const threadSelectors =
+      threadIndex !== null && getThreadSelectors(threadIndex);
+
+    return {
+      rightClickedCallNode:
```

Comment:
This was done a little bit differently in the original PR, but the more I thought about it the more it made sense to completely switch call node context menu to use the same thread as the right-clicked call node has instead of mixing selected thread with right clicked call node (even though currently they probably always be the same)